desktop/window-handlers/external-links/index.js

```diff
@@ -9,7 +9,7 @@ const debug = require( 'debug' )( 'desktop:external-links' );
 /**
  * Internal dependencies
  */
-const Config = require( 'lib/config' );
+const Config = require( '../../lib/config' );
```

Comment:
There are some many places that need to use this strange construct. Maybe we should move this folder elsewhere or rename it to something else? :D